src/index.html

```diff
@@ -11,8 +11,66 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Space+Grotesk:wght@300..700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__picture">
+        <img
+          src="images/imac.jpeg"
+          alt="IMac"
+          class="picture"
+        />
+      </div>
+
+      <div class="card__info">
+        <div class="text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="code">Product code: 195434</div>
+      </div>
+
+      <div class="card__rating">
+        <div class="stars">
```

Comment:
Add the modifier classes
```suggestion
        <div class="stars stars--4">
```